src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6275,7 +6275,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
     <value>The receiver of a `with` expression must have a non-void type.</value>
   </data>
   <data name="ERR_NoSingleCloneMethod" xml:space="preserve">
```

Comment:
>ERR_NoSingleCloneMethod [](start = 14, length = 23)

Consider renaming the error to more accurately reflect its purpose/conditions under which it is reported